client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { CountComparisonCard } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+
+function SubscribersOverviewCardStats() {
+	const overviewCardStats = [
+		{
+			heading: translate( 'Today' ),
+			count: 1410,
+		},
+		{
+			heading: translate( '30 days ago' ),
+			count: 1080,
+			previousCount: 1080,
+		},
+		{
+			heading: translate( '60 days ago' ),
+			count: 1000,
+			previousCount: 1080,
+		},
+		{
+			heading: translate( '90 days ago' ),
+			count: 970,
+			previousCount: 1080,
+		},
+	];
+	return overviewCardStats;
+}
+
+function SubscribersOverviewCards() {
+	const overviewCardStats = SubscribersOverviewCardStats();
+
+	return (
+		<div className="overview-cards-list">
+			{ overviewCardStats.map( ( overviewCardStat ) => (
+				<CountComparisonCard
+					key={ overviewCardStat.heading }
+					heading={ overviewCardStat.heading }
+					count={ overviewCardStat.count }
+					previousCount={ overviewCardStat.previousCount }
+					showValueTooltip
+					icon="False"
+				/>
+			) ) }
+		</div>
+	);
+}
+
+function SubscribersOverview() {
+	return (
+		<div className="subscribers-overview">
+			<SubscribersOverviewCards />
```

Comment:
Since this component only has the wrapper div I wouldn't create another component for the cards. 
When you add a mechanism to get the data it will probably happen here so placing the cards here will save you from passing the data as a property.